docs/03-contributing/01-getting_started.md

```diff
@@ -35,9 +35,59 @@ To save yourself much time setting up a development server, you can use our Dock
 > [!TIP]
 > Once you are done with your development server and would like to shutdown the Docker containers, hit `Ctrl+C` in your terminal.
 
-If you'd prefer a development setup without using Docker, then continue on to the next sections.
-
-## Initial setup
+If you'd prefer a development setup without using Docker, then continue on the section *Bare metal installation*.
```

Comment:
I think `_` is used more often for italic.. Or better, could we maybe point to the chapter below via a link? 

That would look something like: `[..link text here..](#header-text-here)`